about.html

```diff
@@ -12,53 +12,88 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
+    <header>
+        <div class=topOfAbout>
+        <section class='topOfAboutPage'>
+            <img src="img\lambda-black.png" alt="Lambda Icon;">
+            <nav class=topNavAbout>
+            <a href="">Home</a>
+            <a href="about.html">About</a>
+            </html>Products</a>
+            <a href="">Blog</a>
+            <a href="">Contact</a>
+            </nav>
+        </section>
+        <img src="img\jumbo-about.png" alt="firstImageAbout">
+        </div>
+        </header>
 
+    <div class="container about-page">
+     <section class='topAboutPage'>
+       <div>
+    <h1>About</h1>
+
+    <p class='aboutP'>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+       </div>
+   </section>
+   <div class='strategyImg'>
     <img src="img/about-plan.png" alt="strategy">
+    </div>
+    <section class='StrategySection'>
+      <div class='StrategyContent'>
+    <h2>Strategy</h2>
     
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
+   <p class="strategyP"> Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+    <button>Learn More</button>
+   </section>
+   </div>
+
+<div>
+   <img src="img/about-working.png" alt="strategy">
+  </div>
+    <div class='howWeWorkContent'>
```

Comment:
It may not be necessary to have a section tag inside of a div tag, as the section is the only child